README.md

```diff
@@ -1,6 +1,6 @@
-[![Download](https://api.bintray.com/packages/sealink/maven/printing/images/download.svg) ](https://bintray.com/sealink/maven/printing/_latestVersion)
+[ ![Download](https://img.shields.io/github/v/release/sealink/ticket_printer) ](https://github.com/sealink/ticket_printer/packages/961689)
```

Comment:
I thought these weren't supposed to have whitespace in them? None of the others do